spark/src/main/scala/com/alibaba/graphar/reader/VertexReader.scala

```diff
@@ -94,7 +94,11 @@ class VertexReader(prefix: String, vertexInfo: VertexInfo, spark: SparkSession)
     val pg0: PropertyGroup = propertyGroups.get(0)
     val df0 = readVertexPropertyGroup(pg0, false)
     if (len == 1) {
-      return df0
+      if (addIndex) {
+        return IndexGenerator.generateVertexIndexColumn(df0)
```

Comment:
NB: this is fixing the bug that no index column in the result when property group only contain one property.